### DISCUSSION TITLE
Questions about ormar.ForeignKey()

### DISCUSSION THREAD

**Discussion by ProgrammerPlus1998** [collerek/ormar#299] [Q&A]
I read the help document, this method only passes a table name, but the sql table needs to be associated with a certain field of the table.
Does `ormar.ForeignKey()` have a default association?
What if I want to associate a key specified in another table?

**Comment by collerek** [comment]
Yes, as of now relations always lead to the primary key of the related model.
This is one of the reasons (apart from best practice) why each model has to have pk.

That covers 90% of the use cases (well with composite pks (in development) probably even 95-99%), hence I decided to stick with that to simplify the build.
That can be an enhancement proposal. :)
